build/local_builder.go

```
@@ -324,6 +313,65 @@ func (l *LocalBuilder) UpdateImageMetadata() error {
 	return nil
 }
 
+// setClusterFileToImage: set cluster file whatever build type is
+func (l *LocalBuilder) setClusterFileToImage() {
+	var clusterFileData string
+	if utils.IsFileExist(common.RawClusterfile) {
+		bytes, err := utils.ReadAll(common.RawClusterfile)
+		if err != nil {
+			logger.Warn("failed to set cluster file to image metadata,%s not exist", common.RawClusterfile)
```

do not ignore the err.

---

better return the err